net/mosquitto/Portfile

```diff
@@ -4,94 +4,54 @@ PortSystem          1.0
 PortGroup           cmake 1.1
 
 name                mosquitto
-version             2.0.9
-revision            1
+version             2.0.20
+revision            0
 
 categories          net devel
-platforms           darwin
 license             {EPL-1 EDL-1}
 
 maintainers         {gmail.com:slewsys @slewsys} openmaintainer
 
 description         Mosquitto is an open-source MQTT 3.1/3.1.1 broker
 
 long_description    \
-    MQTT is a machine-to-machine / "Internet of Things" connectivity protocol. \
-    Mosquito provides an open-source MQTT v3.1.1 broker (i.e., server) and \
-    both  C and C++ client libraries.
+    MQTT is an OASIS standard messaging protocol for the Internet of Things (IoT). \
+    Mosquitto is an open source implementation of a server for version 5.0, 3.1.1, \
+    and 3.1 of the MQTT protocol. It also includes a C and C++ client library, and \
+    the mosquitto_pub and mosquitto_sub utilities for publishing and subscribing.
 
 homepage            https://mosquitto.org
-master_sites        http://mosquitto.org/files/source/
+master_sites        ${homepage}/files/source/
 
-checksums           rmd160  6925455f51b6c79ebd160aba2f3709864aa5c05d \
-                    sha256  1b8553ef64a1cf5e4f4cfbe098330ae612adccd3d37f35b2db6f6fab501b01d4 \
-                    size    757878
+checksums           rmd160  0e030b778f914da49b8436ef3d635090e120e1d3 \
+                    sha256  ebd07d89d2a446a7f74100ad51272e4a8bf300b61634a7812e19f068f2759de8 \
+                    size    799972
 
 depends_build-append \
-                    path:bin/xsltproc:libxslt
+                    path:bin/xsltproc:libxslt \
+                    port:docbook-xsl-nons
 
 depends_lib         port:c-ares \
+                    port:libcjson \
                     port:libwebsockets \
                     port:tcp_wrappers \
                     path:lib/libssl.dylib:openssl
 
 depends_test-append \
-                    port:python27
+                    port:cunit \
+                    port:python312
 
 configure.args-append \
                     -DUSE_LIBWRAP:BOOL=ON \
                     -DWITH_SRV:BOOL=ON \
                     -DWITH_WEBSOCKETS:BOOL=ON
 
-test.run            yes
-test.target         -C ${build.dir}/test test
+test.run            no
```

Comment:
```suggestion
```

Reply:
no test-phase is the default; also if you don't run the tests please just remove the test dependencies